packages/slice-machine/test/pages/slices.test.tsx

```diff
@@ -351,9 +352,14 @@ describe("slices", () => {
       fireEvent.change(nameInput, { target: { value: "FooBar" } });
     });
 
-    const submitButton = screen.getByText("Create");
+    const createSliceModal = document.querySelector<HTMLElement>(
```

Comment:
⚠ Why using data-cy instead of getByText? We should try removing dependencies in unit test to Cypress selectors as much as possible.